doc/developer_guide/notebooks.md

```diff
@@ -14,6 +14,19 @@ e.g. `exasol-notebook-connector @ git+https://github.com/exasol/notebook-connect
 It is important to reference a commit in the `main` branch, not the `main` branch itself. The git dependency should be
 replaced with the proper pypi dependency before releasing of the AI-Lab.
 
+## Work in progress Notebooks
+
+Sometimes, we might have notebooks which we don't want to release yet, either because thy are incomplete or 
+only a pre-release. However, we already want to test these notebooks using the notebook tests or build 
+Docker images that contain them.
+These work in progress notebooks should be located in the work in progress directory.
+
+```exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress```
+
+The work in progress directory then only gets copied into the Docker image when either the test parameter 
```

Comment:
```suggestion
The directory then only gets copied into the Docker image when either the test parameter 
```

Reply:
BTW: What is the "test parameter"? Does it have a name?
How can I (as a user) set this parameter?

Reply:
the user can't set, it is part of the parametrized fixture

Reply:
If the text refers to this parameter, I propose adding a link, e.g. pointing to the fixture.

Reply:
will do